src/widgets/refinement-list/defaultTemplates.tsx

```diff
@@ -0,0 +1,40 @@
+/** @jsx h */
+import { h } from 'preact';
+
+import type { RefinementListComponentTemplates } from './refinement-list';
+import { formatNumber } from '../../lib/formatNumber';
+import { cx } from '../../lib/utils';
+
+const defaultTemplates: RefinementListComponentTemplates = {
+  item({ cssClasses, count, value, highlighted, isRefined, isFromSearch }) {
+    return (
+      <label className={cx(cssClasses.label)}>
+        <input
+          type="checkbox"
+          className={cx(cssClasses.checkbox)}
+          value={value}
+          defaultChecked={isRefined}
+        />
+        <span
+          className={cx(cssClasses.labelText)}
+          dangerouslySetInnerHTML={
+            isFromSearch ? { __html: highlighted } : undefined
+          }
```

Comment:
unless you think that would be too big of a possible breaking change (as it used {{{ before), in which case maybe an  `@MAJOR or @TODO` comment?

Reply:
If we want to use the `<Highlight>` component we need to adapt it because SFFV highlighting uses a different DOM (different class names and didn't have a wrapping tag for non highlighted parts).

I think we can wait for the next major and avoid extra code for now. I added an annotation in 4531f2ad.